src/components/HTMLEngineProvider/HTMLRenderers/CodeRenderer.js

```diff
@@ -16,7 +17,13 @@ function CodeRenderer(props) {
         fontWeight: textStyle.fontWeight,
     });
 
+    // Check if the current element is inside an h1 tag, and set a specific font size if true,
+    // otherwise, use the font size defined in `textStyle`.
+    const fontSize = HTMLEngineUtils.isInsideH1(props.tnode) ? 15 : textStyle.fontSize;
```

Comment:
Styles should not be inlined. Write a style function that takes isInsideH1 as a parameter and returns the font size.